doc/source/conf.py

```diff
@@ -83,11 +83,8 @@ class PyTypeObject(ctypes.Structure):
     'matplotlib.sphinxext.plot_directive',
     'IPython.sphinxext.ipython_console_highlighting',
     'IPython.sphinxext.ipython_directive',
-    'sphinx.ext.imgmath',
```

Comment:
For some reason this works, but I don't know if that's an accident or not. Why isn't `sphinx.ext.mathjax` added here?

Reply:
Maybe some markup is automatically rendered if you have mathjax installed on your browser? I can't see anything about `sphinx.ext.mathjax` being default in any sphinx version, I think we should use it.

Reply:
I also think the submodule is needed for offline use. I note the scipy module is 4 years old and mathjax has been recently updated. I'm going to push this off to 1.22 until we understand what is going on. We can backport it if needed.

Reply:
One thing we might want to check is browser compatibility for MathJax 3.* if we use it.
 > In particular, to allow MathJax version 3 to work with IE11, include the line
`<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>`

Reply:
> We can backport it if needed.

I don't think that is needed. The most important benefit here is to get rid of the LaTeX requirement for local doc builds and in Gitpod - both of which are not quite relevant on a release branch.

Reply:
Updating MathJAX to a more recent release could maybe give some rendering performance gains. There's no bugs or other issues in the current version AFAIK.

Reply:
We don't use any of the options except for `mathjax_path`, which points to the `scipy-mathjax` submodule. So I think there's nothing Sphinx can mess up for us here.